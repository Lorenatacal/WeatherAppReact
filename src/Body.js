import React, { Component } from 'react';
import axios from 'axios';
import Weather from './Weather';
import FahrenheitButton from './FahrenheitButton';

class Body extends React.Component {
    render () {
        return (
            <div>
                <Weather />
                <FahrenheitButton />
            </div>
        )
    }
}

export default Body;