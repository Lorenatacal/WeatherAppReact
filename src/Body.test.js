import React, { Component } from 'react';
import Body from './Body';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import { ENETRESET } from 'constants';

Enzyme.configure({ adapter: new Adapter() });

test("Body should render correctly", () => {
    const wrapper = Enzyme.shallow(<Body />)
    expect(toJson(wrapper)).toMatchSnapshot();
});