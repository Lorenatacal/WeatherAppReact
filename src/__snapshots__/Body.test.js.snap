// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body should render correctly 1`] = `
<div>
  <Geolocated(Weather) />
</div>
`;
